config/routes.rb

```diff
@@ -0,0 +1,7 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+  get '/messages/', to: 'messages#index'
+end
```

Comment:
- [ ] also make sure that you have the right call for the messages, I think the get => to is not the right one. it will be simple just to call root or resource.

Reply:
The API endpoints it's working that way. But I will also add the endpoint to the root if that is necessary.

Reply:
Yes, I didn't db:seed because the readme file was vage.

Reply:
Please add details to the readme file about the seed and wich localhost for rails and react to display the messages.